boards/arm/reel_board/doc/reel_board.rst

```diff
@@ -0,0 +1,434 @@
+
++-------+-----------------------+---------------------------+
+| Name  | Type                  | Usage                     |
++=======+=======================+===========================+
+| J19   | closed solder jumper  | testpoint I2C SDA         |
++-------+-----------------------+---------------------------+
+| J20   | closed solder jumper  | testpoint I2C SCL         |
++-------+-----------------------+---------------------------+
+| J7    | closed solder jumper  | testpoint INT1 MMA8652    |
++-------+-----------------------+---------------------------+
+| J11   | closed solder jumper  | testpoint INT APDS9960    |
++-------+-----------------------+---------------------------+
+| J12   | closed solder jumper  | testpoint DRDYn HDC1010   |
++-------+-----------------------+---------------------------+
+
+EPD testpoints
+--------------
+
++-------+-----------------------+---------------------------+
+| Name  | Type                  | Usage                     |
++=======+=======================+===========================+
+| J13   | closed solder jumper  | testpoint EPD Busy        |
++-------+-----------------------+---------------------------+
+| J14   | closed solder jumper  | testpoint EPD Reset       |
++-------+-----------------------+---------------------------+
+| J15   | closed solder jumper  | testpoint EPD DC          |
++-------+-----------------------+---------------------------+
+| J16   | closed solder jumper  | testpoint EPD SPI_CS      |
++-------+-----------------------+---------------------------+
+| J17   | closed solder jumper  | testpoint EPD SPI_CLK     |
++-------+-----------------------+---------------------------+
+| J18   | closed solder jumper  | testpoint EPD SPI_MOSI    |
++-------+-----------------------+---------------------------+
+
+Power supply testpoint
+----------------------
+
++-------+-----------------------+-------------------------------------------+
+| Name  | Type                  | Usage                                     |
++=======+=======================+===========================================+
+| J21   | closed solder jumper  | testpoint peripheral voltage on/off       |
++-------+-----------------------+-------------------------------------------+
+| TP11  | testpoint             | testpoint peripheral voltage              |
++-------+-----------------------+-------------------------------------------+
+| TP12  | testpoint             | testpoint nRF52840 supply voltage VDD_nRF |
++-------+-----------------------+-------------------------------------------+
+| TP13  | testpoint             | testpoint boost converter input voltage   |
++-------+-----------------------+-------------------------------------------+
+
+Build-in Debug Adapter
+======================
+
+Debug Adapter is based on the DAPLink interface firmware and
+NXP MK20DX128VFM5 SoC. The adapter is powered via micro USB connector and
+is always on when the board is connected to the USB host.
```

Comment:
Does this imply that the debug adapter is NOT functional if the board is powered using the on-board battery?

Reply:
No, "can be flashed and debugged, powered either from battery or USB"